add docu snipped for draftPrepare with =DraftMessages (#2621)
cap-js/docs: java/fiori-drafts.md

| POST with action `draftNew`                                            | `DraftService.EVENT_DRAFT_NEW`     | Creates a new empty draft. Internally triggers `DRAFT_CREATE`.                                                              |
| PATCH with key `IsActiveEntity=false`                                  | `DraftService.EVENT_DRAFT_PATCH`   | Updates an existing draft.                                                                                                  |
| DELETE with key `IsActiveEntity=false`                                 | `DraftService.EVENT_DRAFT_CANCEL`  | Deletes an existing draft.                                                                                                  |
| DELETE with key `IsActiveEntity=true`                                  | `CqnService.EVENT_DELETE`          | Deletes an active entity *and* the corresponding draft.                                                                     |
| POST with `draftPrepare` action                                        | `DraftService.EVENT_DRAFT_PREPARE` | Empty implementation.                                                                                                       |

> | POST with `draftPrepare` action and `$select=DraftMessages`            | `DraftService.EVENT_CREATE`        | Creates a new active entity including all handlers and validations but ultimately performs a transaction rollback.          | 

Reviewer: ```suggestion
| POST with `draftPrepare` action and `$select=DraftMessages`            | `DraftService.EVENT_DRAFT_PREPARE`        | Simulates the draft activation to ensure complete validation of the document. It triggers a `draftActivate` (internally runs a `CREATE` or `UPDATE` event) in an isolated transaction, which is ultimately rolled back.          | 
```

> | POST with `draftPrepare` action and `$select=DraftMessages`            | `DraftService.EVENT_DRAFT_PREPARE`        | Simulates the draft activation to validate your document completely. The system triggers a `draftActivate` operation (internally runs a `CREATE` or `UPDATE` event) in an isolated transaction and then rolls it back.         | 

Author: @renejeglinsky the hint about the internal create event is pretty relevant and actually the cause for this PR. So we need to keep it in here.

Reviewer 2: It is also documented in line 85, so I misinterpreted the "internally" in that case. So here's a proposal to make it shorter/concise:

```suggestion
| POST with `draftPrepare` action and `$select=DraftMessages`            | `DraftService.EVENT_DRAFT_PREPARE`        | Simulates the draft activation to validate your document completely. The system triggers a `draftActivate` operation (`CREATE` or `UPDATE` event) in an isolated transaction and then rolls it back.        | 
```

> | POST with `draftPrepare` action and `$select=DraftMessages`            | `DraftService.EVENT_DRAFT_PREPARE`        | Simulates the draft activation to ensure complete validation of the document. It triggers a `draftActivate` (internally runs a `CREATE` or `UPDATE` event) in an isolated transaction, which is ultimately rolled back.          | 

Reviewer 2: Why should we mention what is run internally? Can we skip this information here?
```suggestion
| POST with `draftPrepare` action and `$select=DraftMessages`            | `DraftService.EVENT_DRAFT_PREPARE`        | Simulates the draft activation to validate your document completely. The system triggers a `draftActivate` operation in an isolated transaction and then rolls it back.         | 
```
